client/verta/verta/deployment/__init__.py

```diff
@@ -5,3 +5,9 @@
     prediction_input_unpack,
     prediction_io_cleanup,
 )
+
+from .strategies import (
+    _UpdateStrategy,
+    DirectUpdateStrategy,
+    CanaryUpdateStrategy,
+)
```

Comment:
I'd prefer to not do this, and have users import strategies via
```python
from verta.deployment.strategies import ...
```

Reply:
Removed.